app/src/main/res/values-it/strings.xml

```diff
@@ -0,0 +1,224 @@
+<resources>
+    <string name="app_name">GreenStash</string>
+
+    <!-- Other Strings -->
+    <string name="confirm">Conferma</string>
+    <string name="cancel">Cancella</string>
+    <string name="OK">OK</string>
```

Comment:
Sorry i meant to change the value of the string to `OK` from `oK` bot the name

Reply:
`<string name="ok">OK</string>`

Reply:
done

Reply:
oh wait "ok" not "oK"? my bad